ajax/libs/emojione/3.1.6/css/emojione-awesome.css

```diff
@@ -0,0 +1,8026 @@
+[class*=e1a-] {
+  display: inline-block;
+  width: 1em;
+  height: 1em;
+  margin: -.2ex .05em 0 .1em;
+  line-height: normal;
+  vertical-align: middle;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: 1em 1em; }
+
+.e1a-sm {
+  width: 32px;
+  height: 32px;
+  margin: -0.2ex 0.05em 0 0.1em;
+  background-size: 1em 1em; }
+
+.e1a-med {
+  width: 64px;
+  height: 64px;
+  margin: -0.4ex 0.1em 0 0.2em;
+  background-size: 2em 2em; }
+
+.e1a-lg {
+  width: 128px;
+  height: 128px;
+  margin: -0.8ex 0.2em 0 0.4em;
+  background-size: 4em 4em; }
+
+.e1a-interrobang {
+  background-image: url("https://cdn.jsdelivr.net/emojione/assets/3.1/png/128/2049.png"); }
```

Comment:
I have concern here, this may confuse users and even make the usage more complex, maybe should contact upstream for builds without the absolute path, but host locally, or even more, just like jsdelivr, to have a cdnjs build version.